spec/syntax.md

```diff
@@ -397,9 +394,9 @@ Additionally, an _input-declaration_ can contain a _variable-expression_.
 > Placeholders:
 >
 > ```
-> {This placeholder contains an {|expression with a literal|}}
-> {This placeholder references a {$variable}}
-> {This placeholder references a function on a variable: {$variable :function with=options}}
+> {{This placeholder contains an {|expression with a literal|}}}
+> {{This placeholder references a {$variable}}}
+> {{This placeholder references a function on a variable: {$variable :function with=options}}}
```

Comment:
No need to quote these patterns; better keep them as valid messages.

```suggestion
> This placeholder contains an {|expression with a literal|}
> This placeholder references a {$variable}
> This placeholder references a function on a variable: {$variable :function with=options}
```

Reply:
Ugh. I thought I got this one...